src/test/scala/chiselTests/properties/PropertySpec.scala

```diff
@@ -213,13 +213,14 @@ class PropertySpec extends ChiselFlatSpec with MatchesAndOmits {
   }
 
   it should "fail to compile when connecting Property types of different types" in {
```

Comment:
Fix test name, in that it should pass?

Reply:
It actually fails when using connectable operators so I'm just turning the `:=` into a `:#=` and restoring the failing test.

The reason for the change is that before `Property` was `Data`, it had it's own `:=` which was type safe. Now that it's a `Data`, `:=` is not type safe, but connectable operators are so we have to use them to illustrate that behavior.